expenseease-group-svc/src/main/java/com/expenseease/group/model/ErrorResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.expenseease.group.model;
+
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+public class ErrorResponse {
+    private Integer status;
+    private String message;
+    private Long timeStamp;
+
+}
```

Comment:
We already have a class for generic response, use the same.
Refer to ExpenseEaseResponse class

Reply:
This is present in IAM service, copy the same to Group service.

Reply:
ExpenseEaseResponse class in different project How do I add this project in this current project as dependency?